forte/data/data_pack.py

```diff
@@ -1426,8 +1504,9 @@ def in_span(self, inner_entry: Union[int, Entry], span: Span) -> bool:
         r"""Check whether the ``inner entry`` is within the given ``span``. The
         criterion are as followed:
 
-        Annotation entries: they are considered in a span if the begin is not
-        smaller than `span.begin` and the end is not larger than `span.end`.
+        Annotation/AudioAnnotation entries: they are considered in a span if the
+        begin is not smaller than `span.begin` and the end is not larger than
+        `span.end`.
```

Comment:
notice `Span` type does not have a unit associated with it. That means if we do `in_span(audio_annotation, text_annotation.span)`, the function actually runs, but this is not what we want. could this be a potential issue?